packages/inter-protocol/src/stakeFactory/stakeFactory.js

```diff
@@ -209,7 +209,7 @@ export const start = async (
   const publicFacet = augmentPublicFacet(
     Far('stakeFactory public', {
       makeLoanInvitation: () =>
-        zcf.makeInvitation(offerHandler, 'make RUNstake'),
+        zcf.makeInvitation(offerHandler, 'make stakeFactory'),
```

Comment:
Good catch. This change is part of the visible contract API, so it's more important than many of the others. Please consider separating it out as a separate commit marked as a breaking change fix.

Reply:
Visible to end users but not read by machines, correct? I.e. I don't believe this change breaks any commitments of the API.
https://github.com/Agoric/agoric-sdk/blob/master/packages/zoe/src/contractFacet/types.js#L93-L97

Reply:
I think it would make sense for a machine to select among invitations from an installation based on description. This is discussed in #2230 for example:

> Just knowing the instance and description should be enough to choose an invitation to use.